contracts/teller-network/Escrow.sol

```diff
@@ -352,7 +354,7 @@ contract Escrow is Pausable, MessageSigned, Fees, Arbitrable {
         require(_escrowId < transactions.length, INVALID_ESCROW_ID);
```

Comment:
I see this missed the check so that only the seller can do it. Or maybe we change the cancel function so that it transfers to the seller all the time and not the `msg.sender`

Reply:
good catch, dude!

Reply:
I'm an auditor now 🕵